### PR TITLE
rolling-update - initial GCE support

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

**Comment by justinsb** [comment]
cc @chrislovecnm

File: upup/pkg/fi/cloudup/gce/instancegroups.go [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+		}
+	}
+
+	zones, err := c.Zones()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, zoneName := range zones {
+		err := c.Compute().InstanceGroupManagers.List(project, zoneName).Pages(ctx, func(page *compute.InstanceGroupManagerList) error {
```
Comment (chrislovecnm) [inline upup/pkg/fi/cloudup/gce/instancegroups.go]: Can we filter this at all?

**Comment by justinsb** [comment]
Rebased and tested on AWS

**Comment by chrislovecnm** [comment]
Tested on GCE?

**Comment by justinsb** [comment]
Yes, tested on GCE :-)

**Comment by chrislovecnm** [comment]
Added to 1.8.0, but is that too soon?

**Comment by justinsb** [comment]
No, I think we should get it in (with suitable testing of course), because these structural changes are going to make backports complicated if we don't get them in now...

**Pull request merged**